RedHatInsights/vulnerability-ui
# feat(custom report): Use radio buttons for sort selection in dropdown

https://issues.redhat.com/browse/RHINENG-12947

I was able to repurpose the existing `RadioCustomFilter` component which is already used for filters inside the same modal allowing me to remove the redundant `SelectCustomSorter` component.

## Before
![Screenshot from 2024-09-29 19-59-06](https://github.com/user-attachments/assets/c0a49584-5041-4b9f-a360-b1a0dd05032d)


## After
![Screenshot from 2024-09-29 19-58-51](https://github.com/user-attachments/assets/17ee6aa4-56bc-48d1-bc6c-cc482ebdcd88)

## To test
1. Navigate to `Reports` page
2. Open custom report modal with `Create report` button
3. Try to export PDF with the default selected sort
4. Export PDF and check if CVEs are sorted properly (CVSS high to low)
5. Open modal again and change the sorting
6. Export PDF and check if CVEs are sorted properly
